library/core/src/slice/index.rs

```diff
@@ -92,27 +93,27 @@ const fn slice_index_order_fail(index: usize, end: usize) -> ! {
 }
 
 // FIXME const-hack
+#[inline]
```

Comment:
Hm, I'm concerned about changing this for all cases. Right now this not being inlined is good for code-size in the non-`panic_immediate_abort` case.

Reply:
Hmmm... yeah, I see -- since inlining is bottom up, this should be inlined into `slice_index_order_fail`, which will presumably not be inlined.

That makes a reasonable amount of sense, but I'll do a perf run to check.